src/_locales/en/messages.json

```diff
@@ -306,7 +306,7 @@
     "description": "Question #7"
   },
   "faqAnswer7": {
-    "message": "The original author of this extension is David Šabata. Now this extension is developed and maintained by so-called Web Scrobbler Team.",
+    "message": "The original author of this extension is David Šabata. Now this extension is developed and maintained by the so-called Web Scrobbler Team.",
```

Comment:
I would even go so far as to drop the `so-called` part - that doesn't read well IMHO